### DISCUSSION TITLE
How do you request REST API?

### DISCUSSION THREAD

**Discussion by aminick** [supabase/supabase#860] [Questions] [answered]
Keep getting 
```
{
    "message": "no Route matched with those values"
}
```

My request url is `<supabase_api_url>/<table_name>`

**Comment by soedirgo** [comment, marked as answer]
Hi @aminick! If you want to interact with the REST endpoint directly, you can do something like this:
```
GET https://your_project_ref.supabase.co/rest/v1/your_table_name?select=*
apikey: your.jwt.apikey
```

**Reply by kiwicopple** [reply]
Note also 

> apikey: your.jwt.apikey

This should be the anon key, which you can find in Settings > API

**Reply by yamin8000** [reply]
> 
> 
> Hi @aminick! If you want to interact with the REST endpoint directly, you can do something like this:
> 
> ```
> GET https://your_project_ref.supabase.co/rest/v1/your_table_name?select=*
> apikey: your.jwt.apikey
> ```

How can I find references for more complex queries like filters?
[Official references](https://supabase.io/docs/reference/javascript/using-filters) are mostly for java script, for pure rest requests I can't find good reference.

**Reply by soedirgo** [reply]
@yamin8000 for docs on the syntax for REST, you can try the [PostgREST docs](https://postgrest.org/en/stable/api.html#horizontal-filtering-rows).
